Documentation/subcommands/stop.md

```diff
@@ -0,0 +1,32 @@
+# rkt stop
+
+Given a list of pod UUIDs, rkt stop will shut them down, for the shipped stage1 images, this means:
+
+* default systemd-nspawn stage1: the apps in the pod receive a TERM signal and, after a timeout, a KILL signal.
+* kvm stage1: the virtual machine is shut down with `systemctl halt`.
+* rkt fly stage1: the app receives a TERM signal.
+
+The `--force` flag will stop a pod forcibly, that is:
+
+* default systemd-nspawn stage1: the container is killed.
```

Comment:
FYI: this was original @iaguis idea introduced [there](https://github.com/coreos/rkt/pull/1959/commits/86c685bc20b26a1c37386c497d61c9ccbba5b1e5).